internal/testing/cluster_test.go

```diff
@@ -267,4 +263,27 @@ func TestEphemeralAccounts(t *testing.T) {
 	if !reflect.DeepEqual(busAcc, acc) {
 		t.Fatal("bus account doesn't match worker account")
 	}
+
+	// Shut down cluster.
+	if err := cluster.Shutdown(context.Background()); err != nil {
+		t.Fatal(err)
+	}
+
+	cluster2, err := newTestClusterWithFunding(dir, false, zap.New(zapcore.NewNopCore()))
```

Comment:
I don't understand what `WithFunding` has to do with the integration test. It seems like we're not particularly asserting something that would be different with or without funding? Probably missing it though but it's not really obvious on first sight.

Reply:
It's just to speed the test up. We are only interested in loading the persistence and checking if it's alright. So we don't need to mine a bunch of blocks again.